src/liquid/components/ld-header/readme.md

```diff
@@ -0,0 +1,306 @@
+      Register
+    </button>
+    <button class="ld-button ld-button--brand-color ld-button--ghost ld-button--sm" id="login-sm" title="Login" type="button">
+      <svg class="ld-icon ld-icon--lg" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9315 2C9.4767 2 7.455 3.6606 7.455 6.8376c0 2.0938.9386 4.1877 2.2383 5.4873.4332 1.1552-.4332 1.8051-.5776 1.8773-2.5993.9386-5.6318 2.6714-5.6318 4.3321v.6498c0 2.2383 4.4043 2.7437 8.4476 2.8159 4.1156-.0722 8.5199-.5776 8.5199-2.8159v-.6498c0-1.6607-3.0325-3.3935-5.6318-4.3321-.2166-.0722-1.083-.7221-.5776-1.8773 1.2996-1.2996 2.2383-3.3935 2.2383-5.4873C16.4803 3.6606 14.3864 2 11.9315 2z" fill="currentcolor"/></svg>
+    </button>
+    <button class="ld-button ld-button--brand-color ld-button--secondary ld-button--sm" id="login-lg" type="button">
+      <svg class="ld-icon ld-icon--sm" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9315 2C9.4767 2 7.455 3.6606 7.455 6.8376c0 2.0938.9386 4.1877 2.2383 5.4873.4332 1.1552-.4332 1.8051-.5776 1.8773-2.5993.9386-5.6318 2.6714-5.6318 4.3321v.6498c0 2.2383 4.4043 2.7437 8.4476 2.8159 4.1156-.0722 8.5199-.5776 8.5199-2.8159v-.6498c0-1.6607-3.0325-3.3935-5.6318-4.3321-.2166-.0722-1.083-.7221-.5776-1.8773 1.2996-1.2996 2.2383-3.3935 2.2383-5.4873C16.4803 3.6606 14.3864 2 11.9315 2z" fill="currentcolor"/></svg>
+      Login
+    </button>
+  </div>
+</header>
+
+<style>
+  .logo {
+    color: var(--ld-thm-warning);
+    margin-top: calc(var(--ld-sp-8) * -1);
+  }
+
+  #register .ld-icon {
+    display: none;
+  }
+
+  #login-sm {
+    --ld-button-padding-x-sm: 0;
+    --ld-button-padding-y-sm: 0;
+    margin: 0 calc(var(--ld-sp-8) * -1) 0 var(--ld-sp-8);
+  }
+
+  #login-lg {
+    display: none;
+  }
+
+  @media (min-width: 52rem) {
+    #register .ld-icon {
+      display: block;
+    }
+
+    #login-sm {
+      display: none;
+    }
+
+    #login-lg {
+      display: grid;
+    }
+  }
+</style>
+{% endexample %}
+
+### With burger menu button
+
+{% example %}
+<ld-header>
+  <ld-button mode="ghost" type="button">
+    <ld-icon name="burger-menu"></ld-icon>
+  </ld-button>
+  <a href="#" title="Home">
+    <ld-typo class="logo" tag="div" variant="b6">M</ld-typo>
+  </a>
+  <ld-typo tag="div" variant="h5">Liquid Oxygen</ld-typo>
+</ld-header>
+
+<style>
+  .logo {
+    color: var(--ld-thm-warning);
+    margin-top: calc(var(--ld-sp-8) * -1);
+  }
+</style>
+
+<!-- CSS component -->
+
+<header class="ld-header">
+  <div class="ld-header__container">
+    <button class="ld-button ld-button--brand-color ld-button--ghost ld-button--sm" id="burger-menu" title="Login" type="button">
```

Comment:
I really would like to not make the example even more complicated by adding an svg instead of using the predefined icon in the `ld-icon` component. Also this is only an example. The examples don't need to be perfectly a11y, imho, especially as this example focuses on the header and not on the components used inside the header. I can still simply remove the `title` attribute.

Reply:
> The examples don't need to be perfectly a11y

I disagree. We should lead by example.

Reply:
Added `aria-label` instead, same reason as above. Ok?